cmake/ActsFunctions.cmake

```diff
@@ -1,3 +1,22 @@
+# This function creates a library with Acts namespace support
+# It takes the library name without the "Acts" prefix and creates:
+# 1. A direct CMake target named "Acts${library_name}"
+# 2. An ALIAS target named "Acts::${library_name}"
+function(acts_add_library library_name)
+    set(full_target_name "Acts${library_name}")
+
+    # Forward all arguments except the first (library_name) to add_library
+    add_library(${full_target_name} ${ARGN})
+
+    # Create alias target with Acts:: namespace
+    add_library(Acts::${library_name} ALIAS ${full_target_name})
+
+    set_target_properties(
+        ${full_target_name}
+        PROPERTIES EXPORT_NAME Acts::${library_name}
+    )
+endfunction()
```

Comment:
🤔 I do think it would make sense to move all the library related `install(...)` commands in here.